crawl-ref/source/dat/jobs/abyssal-knight.yaml

```diff
@@ -0,0 +1,25 @@
+enum: JOB_ABYSSAL_KNIGHT
+name: "Abyssal Knight"
+short_name: AK
+#category: Zealot
+#category_priority: 90
+str: 4
+int: 4
+dex: 4
+equipment:
+  - "leather armour"
+weapon_choice: WCHOICE_PLAIN
+recommended_species:
+  - SP_HILL_ORC
```

Comment:
Since species are data as well, it would be nice to refer to species by their name, eg `- Hill Orc` instead of exposing the enum.

Reply:
Yep, that's on my todo list.